docs/management/admin/response-actions.asciidoc

```diff
@@ -111,6 +111,29 @@ TIP: You can use the <<use-osquery,Osquery manager integration>> to query a host
 When {elastic-defend} prevents file activity due to <<malware-protection,malware prevention>>, the file is quarantined on the host and a malware prevention alert is created. To retrieve this file with `get-file`, copy the path from the alert's *Quarantined file path* field (`file.Ext.quarantine_path`), which appears under *Highlighted fields* in the alert details flyout. Then paste the value into the `--path` parameter. 
 ====
 
+=== `execute`
+
+Run a shell command on the host. The command's output and any errors are returned in the response console, up to 2000 characters. The complete output (stdout and stderr) are also saved to a downloadable .zip archive (password: `elastic`). Use these parameters:
```

Comment:
This says it's downloadable, so it would be like any other file you download using a browser. It's also probably pretty clear to the user actually performing the actions, because the browser will either ask the user where to save it, or save it to the Downloads folder by default.

Reply:
```suggestion
Run a shell command on the host. The command's output and any errors are returned in the response console, up to 2000 characters. The complete output (stdout and stderr) are also saved to a downloadable `.zip` archive (password: `elastic`). Use these parameters:
```

Reply:
```suggestion
Run a shell command on the host. The command's output and any errors appear in the response console, up to 2000 characters. The complete output (stdout and stderr) are also saved to a downloadable `.zip` archive (password: `elastic`). Use these parameters:
```